src/test/com/microsoft/azure/documentdb/changefeedprocessor/Sample.java

```diff
@@ -50,7 +72,7 @@ public static void main (String[]args) {
         ChangeFeedEventHost host = new ChangeFeedEventHost("hostname", docInfo, docAux, options, hostOptions );
 
         try {
-            host.registerObserver(TestChangeFeedObserver.class);
+            host.registerObserver(com.microsoft.azure.documentdb.changefeedprocessor.internal.TestChangeFeedObserver.class);
```

Comment:
you don't need to make this fully qualified

Reply:
this was result of a refactoring. fixed